CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 - Updated layers:
   - "Earthquakes" updated to include 2021 data
+  - "Monthly Mean Concentration (25km) Feb or March (max monthly extent)" 
```

Comment:
One last suggestion: I'd place a `/` in between "Monthly Mean Concentration (25km)" and "Feb or March (max monthly extent)". I would also probably include the root group "Sea ice":
> "Sea ice / Monthly Mean Concentration (25km) / Feb or March (max monthly extent)" updated to include 2021 data"

Sorry to nitpick!